public/app/components/register/register.svc.js

```diff
@@ -0,0 +1,92 @@
+(function() {
+
+	'use strict';
+
+	angular
+		.module('iserveu')
+		.factory('registerService', [
+			'$rootScope',
+			'$timeout',
+			'authResource',
+			'ToastMessage',
+			'utils',
+			'redirectService',
+			'loginService',
+      'localStorageManager',
```

Comment:
Yeah I've just turned on JS linting for the project, but its a mix of tabs and spaces in most files